react/more_react_concepts/refs_and_memoization.md

```diff
@@ -319,4 +319,4 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
```

Comment:
```suggestion
This section contains helpful links to related content. It isn't required, so consider it supplemental.
```
Literally just fixing a strange alternative apostrophe character to appease the linter.